binary/grakn.bat

```diff
@@ -35,7 +35,8 @@ if "%1" == "" goto missingargument
 
 if "%1" == "console" goto startconsole
 
-if "%1" == "server" goto startserver
+if "%1" == "server"  goto startserver
```

Comment:
When are we adding the debug flag in Windows?

Reply:
I'll raise an issue.

Reply:
https://github.com/graknlabs/common/issues/37